### DISCUSSION TITLE
SimNIBS virtual environment

### DISCUSSION THREAD

**Discussion by annika-mackenzie** [simnibs/simnibs#216] [6. Getting started]
Hello,

We are attempting to set up SimNIBS on a high performance computing cluster which does not allow Anaconda environments. While attempting to convert the conda environment to a virtual environment I have had difficulty installing some of the required packages. 

Does anyone have experience setting up a virtual environment for SimNIBS?

Thank you!

**Comment by oulap** [comment]
If you use the installer you shouldn't need conda because all the necessary packages are packed inside the installer. 

You can then use the simnibs environment by calling "simnibs_python" which sees all the necessary packages.

If that's not an option, could you specify what you mean by Anaconda environments? You're not allowed to install packages from the conda channels, or you can't use conda as the virtual environment manager?
